app/formSchema/uiSchema/uiSchema.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  acknowledgements,
+  alternateContact,
+  authorizedContact,
+  benefits,
+  budgetDetails,
+  contactInformation,
+  coverage,
+  estimatedProjectEmployment,
+  existingNetworkCoverage,
+  organizationLocation,
+  organizationProfile,
+  otherFundingSources,
+  projectArea,
+  projectInformation,
+  projectFunding,
+  projectPlan,
+  submission,
+  supportingDocuments,
+  review,
+  techSolution,
+  templateUploads,
+} from './pages';
+
+const uiSchema = {
+  'ui:order': [
```

Comment:
This should just be the order of the pages

Reply:
Okay take another look when you have time. Our testing caught a couple of bugs and the Happo screenshot testing helped me fix the missing `ui:field` in the review page.